libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './Ingress.module.scss';
+
+interface IngressProps {
+  children: React.ReactNode;  
+}
+
+export const Ingress: React.FC<IngressProps> = ({ children }) => {
+  return (
+    <div className={styles.ingress}>
```

Comment:
Should we return something other than a div here? 'p' maybe?

Reply:
A <div> is a block. Which means it does not work as text. If you place a ingress inside a <p> tag with another piece of text, it would not be on the same line. The div block would not be on the same line as the rest of the text.

Consider using a span instead. A <p> tag is also a bit limiting, as it could not be used inside another <p>-tag. As that is semantically not valid. 

Alternatively the .ingress style in css could use display: inline, then the div would work as a span.